src/localforage.js

```diff
@@ -116,7 +116,7 @@ class LocalForage {
         this._dbInfo = null;
 
         this._wrapLibraryMethodsWithReady();
-        this.setDriver(this._config.driver);
+        this.setDriver(this._config.driver).catch(() => {});
```

Comment:
I was thinking of that, but they are going to get the error at the moment they try to use any localforage method (the "no driver tests" still pass).
So for the developer:
* nothing actually changes with this PR
* localForage seems to do nothing on import, since the error during the default instance initialization is silenced and appears only when a method is used. After 3 duplicate tickets on this, I'm now starting thinking of this as a benefit.